José Pablo digital tools 3

jpgarcia2023 opened the pull request:
Submitting Digital Tools Assignment #3

jpgarcia2023 committed:
Add files via upload

Digital Tools Assignment #3

jpgarcia2023 committed:
Rename 2020-03-29-Michelle Obama in The Daily Princetonian.md to 2021-03-29-Michelle Obama in The Daily Princetonian.md

update type in file name

melissabreynolds commented on _posts/2021-03-29-Michelle Obama in The Daily Princetonian.md:
```diff
@@ -0,0 +1,172 @@
+#### A Newspaper's Physical Characteristics ####
+First, it encodes the article with information about its position and appearance within an issue of the newspaper. This would help a researcher understand what type of content is being recorded. In this case, the content is a front page article reporting on a campus event. The newspaper, however, is also filled with opinion articles, advertisements, and campus announcements among numerous other t ...
+
+Furthermore, an article's position and appearance within an issue can convey some meaning as well since placement on the front page across multiple columns and with a border delineating it may denote some greater importance to the text — at least in the eyes of the newspaper's editors — than if it had been limited to a single column deep into the issue. As such, the article below is encoded with  ...
```
Please go back and change the ``` (triple tick marks) in this paragraph to ` (single tick marks) encasing these tags. Triple tick marks will force a line break and set the code off in its own box; using a single tick mark your code shows up inline with the rest of your text.

melissabreynolds commented on _posts/2021-03-29-Michelle Obama in The Daily Princetonian.md:
````diff
@@ -0,0 +1,172 @@
+Furthermore, an article's position and appearance within an issue can convey some meaning as well since placement on the front page across multiple columns and with a border delineating it may denote some greater importance to the text — at least in the eyes of the newspaper's editors — than if it had been limited to a single column deep into the issue. As such, the article below is encoded with  ...
+
+#### The Information Newspapers Contain ####
+Second, the schema used below encodes the article with tags denoting people's names and of places on the university campus (```<name>```, ```<organization>```, and ```<place>```). The usefulness of these tags is not as obvious in just one article as it would be across a corpus of multiple issues of *The Daily Princetonian* with all their content encoded. However, these tags would be what allows f ...
````
Same thing! Change ``` to ` here.

melissabreynolds commented on _posts/2021-03-29-Michelle Obama in The Daily Princetonian.md:
````diff
@@ -0,0 +1,172 @@
+### Multi-Part Pieces of Information ###
+Another question that emerged throughout the process of encoding the text was how to treat pieces of information with multiple parts like names and quotes. For names, *The Daily Princetonian*'s practice of listing class year following the name of any student or alumni mentioned presented the option of ignoring the class year and just tagging the name or of linking the name with the class year. Th ...
+
+Thus, any name followed by a class year was tagged with ```<student>``` as a container for two more tags: ```<name>``` and ```<class>```, the latter having the precise class year as an attribute. Similarly, quotes have multiple parts: the speaker, the actual quoted remark, and often some additional text that helps make sense of the quoted remark. Thus, all these components were tagged with ```<qu ...
````
Same request: change ``` to ` for ease of legibility.

melissabreynolds commented on _posts/2021-03-29-Michelle Obama in The Daily Princetonian.md:
````diff
@@ -0,0 +1,172 @@
+
+### The Encoded Text ###
+What follows is the encoded text for the article "Fashion show benefits Ethiopian relief fund" in the February 26, 1985 [issue](https://papersofprinceton.princeton.edu/princetonperiodicals/?a=d&d=Princetonian19850226-01.1.1) of *The Daily Princetonian*.
+```
````
This is the correct use of the ``` to set off your code as a block isolated from other text.

melissabreynolds commented on _posts/2021-03-29-Michelle Obama in The Daily Princetonian.md:
````diff
@@ -0,0 +1,172 @@
+```
+<div type="news article">
+  <page num="1">
+    <column num="1-2"><item num="2"><border>
````
My only thought here--and obviously this doesn't matter for the hypothetical markup you've done here--is that I might put `<item num="2">` outside of `<column num="1-2">` because the column number seems to change within the piece while the item number does not. In practice, this might result in interlocking XML tags that wouldn't validate.

melissabreynolds commented on _posts/2021-03-29-Michelle Obama in The Daily Princetonian.md:
```diff
@@ -0,0 +1,172 @@
+        <header>Good choice</header>
+
+        <p>
+          <lastname>Jackson</lastname> came up with the idea of donating receipts from the show to the Relief Fund.
```
Again, not a problem for hypothetical markup, but if this were real and you were attempting to aggregate data, you might want to use a `<person>` tag to identify when the same individuals show up across different articles rather than creating different versions of a `<name>` tag. By separating out `<name>` from `<lastname>` and `<firstname>`, you'd end up with lots of names pulled out of these articles with little ability to designate when the same person is mentioned over and over, as opposed to all the times when the name Obama appears as opposed to Michelle Obama. A `<person>` tag would also help you distinguish between individuals, if for example, there were lots of moments when the last name Smith appeared in The Prince, but this name correlated to a dozen different individuals. Later in the article I see that the writer uses first names for designers, which, again, would be difficult to pair up with the last names you've already tagged unless you had a person tag encasing the `<name>` tags like this: `<person name="Michelle Obama"><fullname>Michelle Obama</fullname></person>`.

melissabreynolds commented on _posts/2021-03-29-Michelle Obama in The Daily Princetonian.md:
```diff
@@ -0,0 +1,172 @@
+
+Name tags could at first be used to find all references to an individual student before going on to identify networks of students based on references in the same article as another student or as campus organizations. Such tags could also reveal clusters of campus life by highlighting what types of things are reported to have happened at various places on the campus. Furthermore, this category of  ...
+
+### Structuring The Text's Hierarchy ###
```
This is a very nice reflection on the tensions between physical description and content description that so often comes up in XML/TEI markup.

melissabreynolds commented on _posts/2021-03-29-Michelle Obama in The Daily Princetonian.md:
```diff
@@ -0,0 +1,172 @@
+
+        <p>For the three women,
+          <quotation>"the best thing to do"</quotation> had developed into almost a full-time commitment over the past semester. They began designing last summer, auditioned models before midterm break and sewed the fifty-plus creations themselves. All three expressed satisfaction with the outcome of their efforts,
+          <quote>with
```
What is your reasoning behind creating the `<quote>` tag to encase the `<quotation>` and `<quoted>` tags? Is it intended to account for indirect speech rather than direct quotation?

melissabreynolds commented on _posts/2021-03-29-Michelle Obama in The Daily Princetonian.md:
```diff
@@ -0,0 +1,172 @@
+        <quote>The presentation
+          <quotation>"gathered up a bunch of dynamic personalities, dynamic models — an all-over dynamic show,"</quotation> said
+          <quoted>
+            <student>
```
You could use the `<student>` tag in the same way I suggested that you use the `<person>` tag above. If you used `<student name="Student Name">` every time a student appears in the article (which it doesn't appear that you've done here, as far as I can tell), you could easily remove the name tags altogether. Probably your imagined researcher won't care to know, literally, how many times the name Michelle appears in The Prince. But they _will_ care about how many times the student Michelle Obama appears. TEI/XML allows you to add the metadata that actually makes the text "Michelle" or "Obama" useful to a researcher beyond a simple text search for every time the last name appears.

melissabreynolds commented on _posts/2021-03-29-Michelle Obama in The Daily Princetonian.md:
```diff
@@ -0,0 +1,172 @@
+### Michelle Obama's Photo ###
+And for the reader who has made it this far, here is the photo from the encoded article that shows Obama modeling at a charity fashion show during her senior year.
+
+![Michelle Obama Fashion Show](https://github.com/jpgarcia2023/hello-world/blob/main/Princetonian_1985-02-26_v109_n018_0001%20photo.png?raw=true)
```
Since this image is really large, why don't you add a `{:width="500px"}` parameter to shrink it a bit just after the parentheses giving the link to the image.

melissabreynolds reviewed (changes requested):
Dear José Pablo,

You've done a very nice job thinking through the physical and qualitative characteristics of this newspaper article, and I'm impressed with the detail of your code. I have made a couple of suggestions here that you might incorporate _if_ you were actually building a digital edition of Prince articles, but by the terms of the assignment, you've more than done enough.

Before I publish the site to the blog (with your permission), I only ask that you change the ``` (triple tick marks) where you use them in the body of your blog post to ` (single tick mark). I've noted the three places where this happens and why it's better to use ` to designate code in paragraphs where you want it to appear inline with the rest of the text. Once those changes are made, I'll approve the pull request and the post will go live.

Grade: A

jpgarcia2023 committed:
Update 2021-03-29-Michelle Obama in The Daily Princetonian.md

I just changed to triple tick marks to single tick marks, and I also updated the image size. If that's all that needs updating, I'm happy to have it published on the site!

jpgarcia2023 commented:
I've made the changes you requested in what I think is the right place on GitHub to do that. If not, please let me know so I can fix it and the post can be published.

melissabreynolds commented:
Dear José Pablo,

I see that you made those changes. All you need to do now is send another pull request and I can merge your post to the site.

MR

From: José Pablo Fernández García ***@***.***>
Date: Tuesday, March 30, 2021 at 1:30 PM
To: HUM-331-Princeton/HUM-331-Princeton.github.io ***@***.***>
Cc: Melissa B. Reynolds <melissa.reynolds@Princeton.EDU>, Comment ***@***.***>
Subject: Re: [HUM-331-Princeton/HUM-331-Princeton.github.io] José Pablo digital tools 3 (#65)

I've made the changes you requested in what I think is the right place on GitHub to do that. If not, please let me know so I can fix it and the post can be published.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/HUM-331-Princeton/HUM-331-Princeton.github.io/pull/65#issuecomment-810443314>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AQUVFL6NLGQI57XIBDVXBXDTGIDAXANCNFSM42BCOP5A>.

jpgarcia2023 commented:
Dear Professor Reynolds 

I’m just getting to this now, and when I went to create a new pull request, the option was only to view the pull request I created last night. It doesn’t currently let me create a new pull request for the existing branch. 

However, I think the existing pull request now also includes the new commits. I think I also have to option of closing the current pull request and then sending a new one. 

I’m summary, I’m very unsure of what I should do on GitHub. 

Best,
José Pablo Fernández García

> On Mar 30, 2021, at 13:46, Melissa Reynolds ***@***.***> wrote:
> 
> ﻿
> Dear José Pablo,
> 
> I see that you made those changes. All you need to do now is send another pull request and I can merge your post to the site.
> 
> MR
> 
> From: José Pablo Fernández García ***@***.***>
> Date: Tuesday, March 30, 2021 at 1:30 PM
> To: HUM-331-Princeton/HUM-331-Princeton.github.io ***@***.***>
> Cc: Melissa B. Reynolds <melissa.reynolds@Princeton.EDU>, Comment ***@***.***>
> Subject: Re: [HUM-331-Princeton/HUM-331-Princeton.github.io] José Pablo digital tools 3 (#65)
> 
> I've made the changes you requested in what I think is the right place on GitHub to do that. If not, please let me know so I can fix it and the post can be published.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub<https://github.com/HUM-331-Princeton/HUM-331-Princeton.github.io/pull/65#issuecomment-810443314>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AQUVFL6NLGQI57XIBDVXBXDTGIDAXANCNFSM42BCOP5A>.
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.

melissabreynolds commented:
Hi José Pablo,

It’s my fault! I didn’t see that your commits had been added to the existing pull request. Nothing else for you to do. You made all the required changes and I’ll merge the post into the main branch.

Best,
MR

On Mar 30, 2021, at 4:41 PM, José Pablo Fernández García ***@***.***> wrote:

﻿

Dear Professor Reynolds

I’m just getting to this now, and when I went to create a new pull request, the option was only to view the pull request I created last night. It doesn’t currently let me create a new pull request for the existing branch.

However, I think the existing pull request now also includes the new commits. I think I also have to option of closing the current pull request and then sending a new one.

I’m summary, I’m very unsure of what I should do on GitHub.

Best,
José Pablo Fernández García

> On Mar 30, 2021, at 13:46, Melissa Reynolds ***@***.***> wrote:
>
> ﻿
> Dear José Pablo,
>
> I see that you made those changes. All you need to do now is send another pull request and I can merge your post to the site.
>
> MR
>
> From: José Pablo Fernández García ***@***.***>
> Date: Tuesday, March 30, 2021 at 1:30 PM
> To: HUM-331-Princeton/HUM-331-Princeton.github.io ***@***.***>
> Cc: Melissa B. Reynolds <melissa.reynolds@Princeton.EDU>, Comment ***@***.***>
> Subject: Re: [HUM-331-Princeton/HUM-331-Princeton.github.io] José Pablo digital tools 3 (#65)
>
> I've made the changes you requested in what I think is the right place on GitHub to do that. If not, please let me know so I can fix it and the post can be published.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub<https://github.com/HUM-331-Princeton/HUM-331-Princeton.github.io/pull/65#issuecomment-810443314>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AQUVFL6NLGQI57XIBDVXBXDTGIDAXANCNFSM42BCOP5A>.
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/HUM-331-Princeton/HUM-331-Princeton.github.io/pull/65#issuecomment-810563187>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AQUVFL2BBKLSEF6IDLSB7C3TGIZN5ANCNFSM42BCOP5A>.
